### PR TITLE
Added Sketchfab icon

### PR DESCRIPTION
**Pull request by moraisandre**
**Issue:** #3458
**Alexa rank:** # 3,861
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [X] I optimized the icon with SVGO or SVGOMG
  - [X] The SVG `viewbox` is `0 0 24 24`

### CODE REVIEW

**Comment by moraisandre** [comment]
I didn't understand why I'm getting the `icon-centered <path> must be centered at (12, 12); the center is currently  (11.995, 12)` error.


I ran the  `npm run svgo -- icons/sketchfab.svg` and I got the following result:

```
> simple-icons@3.5.0 svgo D:\_SourceCode\morais\icon-git\simple-icons
> svgo --config=./.svgo.yml "icons/sketchfab.svg"

sketchfab.svg:
Done in 31 ms!
0.485 KiB - 0% = 0.485 KiB
```

Can anyone help me on this one?

**Comment by NovaGL** [comment]
Try opening the SVG in inkscape it's possible the dimensions are slightly off or try increasing the precision in SVGO

**Comment by moraisandre** [comment]
@NovaGL Tks! Now I've fixed the svglint.

**Comment by moraisandre** [comment]
Is there any issue with my PR?

**Comment by NovaGL** [comment]
Hi  @moraisandre

I'll wait for one of the other maintainers to chime in and review it.

File: _data/simple-icons.json
Comment (runxel) [inline _data/simple-icons.json:5630]: Where have you taken the color from?

Comment (moraisandre) [reply _data/simple-icons.json:5630]: In the press kit page (https://sketchfab.com/press) I downloaded the first image and I got the color from that.

Comment (runxel) [reply _data/simple-icons.json:5630]: Ah okay! Great, then this is good, but please link to this site then instead of root. Thanks!

**Review by runxel — APPROVED** [review]
Perfect, thanks for your work @moraisandre!

**Pull request merged**
